1/SumOfEvenNumbers.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AgileFreaks_test1
+{
+    public class SumOfEvenNumbers
+    {
+        private int s;
+
+        public SumOfEvenNumbers()
+        {
+            s = 0;
```

Comment:
If I would assign 0 value to my instance variable every time my method is
called it should resolve the problem of getting invalid results for
multiple calls.

On Wed, Jun 17, 2015 at 11:57 AM, pirvudoru <notifications@github.com>
wrote:

> In 1/SumOfEvenNumbers.cs
> <https://github.com/Agilefreaks/Test/pull/13#discussion_r32606497>:
>
> > @@ -0,0 +1,36 @@
> > +﻿using System;
> > +using System.Collections.Generic;
> > +using System.Linq;
> > +using System.Text;
> > +using System.Threading.Tasks;
> > +
> > +namespace AgileFreaks_test1
> > +{
> > +    public class SumOfEvenNumbers
> > +    {
> > +        private int s;
> > +
> > +        public SumOfEvenNumbers()
> > +        {
> > +            s = 0;
>
> If you indeed want that sum to be a member on an instance, make the array
> a hard dependency for the instance.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Agilefreaks/Test/pull/13/files#r32606497>.
>